tasks/monitoring.yml

```diff
@@ -14,13 +14,18 @@
   - name: Add repo
     command:
       chdir: /tmp
-      cmd: bash add-monitoring-agent-repo.sh
+      cmd: "bash add-monitoring-agent-repo.sh --also-install --version={{ version }}"
+    environment:
+      CODENAME: "{{ ansible_distribution_release }}"
```

Comment:
Reading through `add-monitoring-agent-repo.sh`, it seems like `REPO_CODENAME` can be used to overwrite the code name that is used to locate the repo, but not `CODENAME`

Taking a step back, what is the `CODENAME` setting for? Inside `add-monitoring-agent-repo.sh`, it uses lsb_release to detect code name so it looks like we do not have to overwrite it from outside the script. Is it here for testing purpose?

Reply:
Good catch, it should be `REPO_CODENAME`.

Igor recommended setting this variable from outside the script, so we wouldn't have to report a change if lsb_release is not installed. See the comment in line 100: https://critique-ng.corp.google.com/cl/352585970/depot/google3/cloud/monitoring/agents/scripts/add-monitoring-agent-repo.sh?left=s13

Reply:
Got it. I guess is controlled by Ansible instead of end users. Is that the case? If so, have we gotten a chance to verify it matches what we expect? aka:

```
OS Family                      REPO_CODENAME
Debian 9 (Stretch)      stretch
Debian 10 (Buster)      buster
Ubuntu 16.04 (Xenial)   xenial
Ubuntu 18.04 (Bionic)   bionic
Ubuntu 20.04 (Focal)    focal
Ubuntu 20.10 (Groovy)   groovy
Amazon AMI              amzn
CentOS / RHEL 7         el7
CentOS / RHEL 8         el8
SLES / SUSE 12          sles12
SLES / SUSE 15          sles15
```

Reply:
<<  I guess {{ ansible_distribution_release }} is controlled by Ansible instead of end users. Is that the case?
Yep.
<<  If so, have we gotten a chance to verify it matches what we expect? 
We only use `REPO_CODENAME` for debian/ubuntu and for those it returns the same result: https://photos.app.goo.gl/p2ATXdrsczh6e9NC7

Reply:
Cool. Thanks for checking!